platform/commonUI/general/test/directives/MCTSplitPaneSpec.js

```diff
@@ -143,7 +143,7 @@ define(
                 });
 
                 it("allows classes to be toggled on contained elements", function () {
-                    controller.toggleClass('resizing');
+                    controller.startResizing('resizing');
```

Comment:
Spec name should be more clear, e.g. "applies class to children when resizing".  You should be testing both `startResizing` and `endResizing`.  Note that `startResizing` does not expect an argument, your test shouldn't pass this as it's misleading.